verifiers/internal/gha/npm.go

```diff
@@ -337,7 +308,26 @@ func verifyPublishPredicateName(att *SignedAttestation, expectedName string) err
 	return nil
 }
 
-func getPublishPredicateData(att *SignedAttestation) (string, string, error) {
+func subjectsFromAttestation(att *SignedAttestation) ([]intoto.Subject, error) {
+	env := att.Envelope
+	pyld, err := base64.StdEncoding.DecodeString(env.Payload)
+	if err != nil {
+		return nil, fmt.Errorf("%w: %s", serrors.ErrorInvalidDssePayload, err.Error())
+	}
+	statement := struct {
+		intoto.StatementHeader
+	}{}
+	if err := json.Unmarshal(pyld, &statement); err != nil {
+		return nil, fmt.Errorf("%w: %s", serrors.ErrorInvalidDssePayload, err.Error())
```

Comment:
nit: wrap both errors
```suggestion
		return nil, fmt.Errorf("%w: %w", serrors.ErrorInvalidDssePayload, err)
```